resources/views/themes/default1/order/index.blade.php

```diff
@@ -296,5 +325,11 @@ function checking(e){
     $('#datepicker4').datepicker({
       autoclose: true
     })
+    $('#datepicker5').datepicker({
```

Comment:
you can use class for such kind of scenarios and can give same class to all of them

Reply:
Fixed